101-azure-machine-learning-service-workspace/azuredeploy.json

```diff
@@ -0,0 +1,134 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "workspaceName": {
+            "type": "string",
+            "defaultValue": "myworkspace",
```

Comment:
does this need to be globally unique?  If so, generate a unique name using uniqueString()